rector.php

```diff
@@ -51,9 +53,8 @@
     $rectorConfig->sets([
         SetList::DEAD_CODE,
         LevelSetList::UP_TO_PHP_74,
-        PHPUnitSetList::PHPUNIT_SPECIFIC_METHOD,
-        PHPUnitSetList::PHPUNIT_80,
```

Comment:
@samsonasik I removed `PHPUNIT_80`. Is it okay? or is it better to keep it?

Reply:
change to `PHPUnitLevelSetList::UP_TO_*` level probably better so no old code will introduced in the future

Reply:
I changed to use `PHPUnitLevelSetList::UP_TO_PHPUNIT_100`.

Reply:
It seems got error on: `system/Test/ControllerResponse.php`, that part seems can be skipped for now, I will try check looking at rector part when I have a chance

Reply:
Current branch update-rector.php is up to date.

Reply:
Thank you, 

- `system/Test/ControllerResponse.php` seems need to be registered to `$rectorConfig->skip()` as path to be checked.
- `tests/system` seems require run against it to apply changes.

Reply:
The following change is not good. `assertObjectNotHasAttribute()` is removed in PHPUnit 10.

```diff
1) tests/system/Config/BaseConfigTest.php:109

    ---------- begin diff ----------
@@ @@
         // override config with shortPrefix ENV var
         $this->assertSame('hubbahubba', $config->delta);
         // incorrect env name should not inject property
-        $this->assertFalse(property_exists($config, 'notthere'));
+        $this->assertObjectNotHasAttribute('notthere', $config);
         // empty ENV var should not affect config setting
         $this->assertSame('pineapple', $config->fruit);
         // non-empty ENV var should overrideconfig setting
    ----------- end diff -----------

Applied rules:
 * AssertPropertyExistsRector
```

Reply:
The following change also not good. The test method has `$this->seeInDatabase()` and it is an assertion.

```
2) tests/system/Database/Live/DeleteTest.php:74

    ---------- begin diff ----------
@@ @@
         $this->seeNumRecords(1, 'user', ['country' => 'US']);
     }

+    /**
+     * @doesNotPerformAssertions
+     */
     public function testDeleteBatch(): void
     {
         $data = [
@@ @@
         $this->dontSeeInDatabase('user', ['email' => 'ahmadinejad@world.com', 'name' => 'Ahmadinejad']);
     }

+    /**
+     * @doesNotPerformAssertions
+     */
     public function testDeleteBatchWithQuery(): void
     {
         $this->forge = Database::forge($this->DBGroup);
    ----------- end diff -----------

Applied rules:
 * AddDoesNotPerformAssertionToNonAssertingTestRector (https://phpunit.readthedocs.io/en/9.5/annotations.html#doesnotperformassertions)
 * RenameClassRector
```

Reply:
@samsonasik Reverted `PHPUnitLevelSetList::UP_TO_PHPUNIT_100` back to `PHPUnitSetList::PHPUNIT_100`.
I don't know why, but it does not work for us. See above.

Reply:
Ok 👍